generator/MechanicsGenerator.ts

```diff
@@ -60,6 +59,19 @@ class MechanicsGenerator {
     return JSON.parse(parsedJson);
     
   }
+  
+  public async getNewCards(cards: CardTemplate[], attackLower: number, attackUpper: number, healthLower: number, healthUpper: number, bossAttackMultiplier: number): Promise<any> {
```

Comment:
in general i like it.  one concern by going completely random is you are losing the spirit of the attack and health the model came up with. i noticed in general it was doing a good job of making low attack / high health  or visa versa.  Also, i found in general it was making more popular characters were more effective.  Instead of this, it might make more sense to do a slightly different algorithm like so.

If it was too easy, then take 3 cards at random and and randomly either add 1 health or attack.  or if it was too hard, do the opposite.

Reply:
bah, let's just commit, i'll make the adjustment, it's super small.

Reply:
Def have the same concerns about losing spirit. Gonna keep playing around and undraft when ready.